src/platform/Kconfig

```diff
@@ -296,17 +296,9 @@ config CAVS_LPS
 	  This option is required to support S0ix/D0ix mode
 	  on cAVS platforms.
 
-config CAVS_LPRO
-	bool "Enable low power ring oscillator as the default clock"
-	default n
-	depends on CAVS && !CAVS_VERSION_1_5
-	help
-	  Select if you want to use 120MHz LPRO as the DSP clock source.
```

Comment:
The description is no longer valid and should be changed.
But are not we able to support down scaled LPRO-only configuration? Could you explain why it is completely removed?

Reply:
@keyonjie  I'll second @mmaka1 that (at least for a while) the ability to select LPRO-only should remain as a debug option. Default configuration shall be dynamic clock switching. I'm OK with dynamic clock switching as the only option on https://github.com/thesofproject/sof/tree/tgl-007-drop-stable and a default one but I'd expect that up-streamed code to master will preserve ability to select LPRO-only support.

Reply:
OK, let me preserve it and rename it to CAVS_LPRO_ONLY.

Reply:
@lbetlej @mmaka1 the refining is here: https://github.com/thesofproject/sof/pull/3205